src/kOS/Communication/kOSConnectivityParameters.cs

```diff
@@ -64,6 +64,8 @@ public override bool HasPresets
             }
         }
 
+        public override string DisplaySection => "kOS";
+
```

Comment:
That's a new feature in C#6, expression bodied property. If I'm not mistaken you need mono version 4 or higher to compile it correctly via monodevelop.